sass/juice.scss

```diff
@@ -7,6 +7,10 @@ body {
   margin: 0;
   box-sizing: border-box;
   background-color: var(--secondary-color);
+  justify-content:space-between;
+  display:flex;
+  flex-direction:column;
```

Comment:
I'd like to keep the `body` less styled as possible, and the `min-height` is enough to fix this. What do you think?

Reply:
I tested locally,the behaviour is the same without flex

Reply:
You are right.  I have another idea to fix this:
```diff
.body {
+  display: flex;
+  flex-direction: column;
+  min-height: 100vh;
}

.main {
+   flex-grow: 1;
}
```

Reply:
@Folyd I did as you suggested